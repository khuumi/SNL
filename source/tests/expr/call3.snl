do func to local a, 7