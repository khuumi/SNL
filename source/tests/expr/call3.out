Call func [(Id Local a), (Int 7)]